tutorials/metrics.md

```diff
@@ -39,22 +39,33 @@ metrics_key = "/path/to/key.pem"
-| `crio_image_pulls_failures`                      | `name`, `error`                                                                                                                         | Counter   | Failed image pulls by image name and their error category.                                           |
-| `crio_image_pulls_layer_size_{sum,count,bucket}` | buckets in byte for layer sizes of 1 KiB, 1 MiB, 10 MiB, 50 MiB, 100 MiB, 200 MiB, 300 MiB, 400 MiB, 500 MiB, 1 GiB, 10 GiB             | Histogram | Bytes transferred by CRI-O image pulls per layer.                                                    |
-| `crio_image_layer_reuse`                         | `name`                                                                                                                                  | Counter   | Reused (not pulled) local image layer count by name.                                                 |
-| `crio_containers_oom_total`                      |                                                                                                                                         | Counter   | Total number of containers killed because they ran out of memory (OOM)                               |
-| `crio_containers_oom`                            | `name`                                                                                                                                  | Counter   | Containers killed because they ran out of memory (OOM) by their name                                 |
-| `crio_processes_defunct`                         |                                                                                                                                         | Gauge     | Total number of defunct processes in the node                                                        |
+| Metric Key                                       | Possible Labels or Buckets                                                                                                              | Type      | Purpose                                                                                                           |
+| ------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------- | --------- | ----------------------------------------------------------------------------------------------------------------- |
+| `crio_operations`                                | every CRI-O RPC\*                                                                                                                       | Counter   | (DEPRECATED) Cumulative number of CRI-O operations by operation type.                                             |
```

Comment:
Let's move the deprecated ones to the end of the table and add which should be used instead.

Reply:
Changed in b1a1bfa.